src/components/pages/deployments/deploymentsGrid/deploymentsGridConfig.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import Config from 'app.config';
+import { TimeRenderer } from 'components/shared/cellRenderers';
+import { gridValueFormatters } from 'components/shared/pcsGrid/pcsGridConfig';
+
+const { checkForEmpty } = gridValueFormatters;
+
+export const deploymentsColumnDefs = {
+  name: {
+    headerName: 'deployments.grid.name',
+    field: 'name',
```

Comment:
Please ensure there is a default sort order in the grid.

Reply:
done